[IMP] Cashbox updates

maq-adhoc opened the pull request.

maq-adhoc committed:
[IMP] Cashbox updates

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/470).

jjscarafia committed:
[REF] cashbox: delete contrainst

por ahora dejamos que se restringa por ir.rule, usuarios con permiso avanzado (accountant) pueden usar las cajas por mas que esten restrictas

de volver a implementarlo hay que ajustarla ya que el difference no estaría evaluando bien (porque puede ser que en una sesion tenga diferencia pero sea un "in valido ")

además evitamos agregar complejidad

jjscarafia committed:
[IMP] cashbox: minor adjustments
1. default company should be env company (not user default one)
2. improove tree view

jjscarafia committed:
[IMP] cashbox: add option to edit dates

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
